x-pack/qa/sql/build.gradle

```diff
@@ -85,6 +88,8 @@ subprojects {
     }
     testCompile "org.elasticsearch.test:framework:${version}"
 
+    testCompile project(path: xpackModule('sql:sql-proto'))
```

Comment:
What about this one?

Reply:
Removed as well (since everything got folded back to qa-sql).